Tensile/SolutionStructs.py

```diff
@@ -3606,7 +3606,11 @@ def assignDerivedParameters(state):
       ldsNumElementsAB = ldsNumElementsAlignedA + ldsNumElementsB
 
     # lds buffer size for reduction
-    ldsNumElementsReduction = state["LocalSplitU"]*state["MacroTile0"]*state["MacroTile1"] if state["LocalSplitU"] > 1 else 0
+    bytesPerComElem = state["ProblemType"]["ComputeDataType"].numBytes()
+    bytesPerLoadElem = state["ProblemType"]["DataType"].numBytes()
+    ratio = bytesPerComElem / bytesPerLoadElem
+
+    ldsNumElementsReduction = ratio*state["LocalSplitU"]*state["MacroTile0"]*state["MacroTile1"] if state["LocalSplitU"] > 1 else 0
```

Comment:
Please convert this type to integer.
It causes an error in CI test.


  File "/var/jenkins_home/workspace/clang_precheckin_Tensile_PR-1613/uj7lkGv2W/tensile/Tensile/KernelWriterAssembly.py", line 3017, in allocateResources

    kStr += inst("s_mov_b32", "m0", hex(kernel["LdsNumElements"] \

TypeError: 'float' object cannot be interpreted as an integer